eng/pipelines/runtime-richnav.yml

```diff
@@ -0,0 +1,43 @@
+trigger:
+  branches:
+    include:
+    - master
+    - dev/infrastructure
+    - release/*.*
+
+pr: none
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+- name: TeamName
```

Comment:
do we need `TeamName`?

Reply:
Nope, you're right. Good catch! Removing